src/storage/exec/EdgeNode.h

```diff
@@ -116,7 +86,28 @@ class FetchEdgeNode final : public EdgeNode<cpp2::EdgeKey> {
                   Expression* exp = nullptr)
         : EdgeNode(context, edgeContext, edgeType, props, expCtx, exp) {}
 
+    bool valid() const override {
+        return valid_;
+    }
+
+    void next() override {
+        valid_ = false;
```

Comment:
Why?

Reply:
They are for fetch vertex/edge, only first data should be used. Previously this job is do in `StorageIterator`, when we change from `prefix` to `get`, there is no more rocksdb iterator.